### ISSUE TITLE
[dataio] test loading data from each season (Trac #1408)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1400]
Grab a few frames from each season, in that season's format, and put them in test data. Then make a test which tries to load each file to make sure we still can.

Migrated from https://code.icecube.wisc.edu/ticket/1408
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T21:51:02", 
    "description": "Grab a few frames from each season, in that season's format, and put them in test data. Then make a test which tries to load each file to make sure we still can.", 
    "reporter": "david.schultz", 
    "cc": "nega, olivas", 
    "resolution": "fixed", 
    "_ts": "1547243462974113", 
    "component": "combo core", 
    "summary": "[dataio] test loading data from each season", 
    "priority": "major", 
    "keywords": "", 
    "time": "2015-10-18T19:36:22", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151018T21:29:55: 
* nega changed owner from "david.schultz" to "nega"
* nega changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151021T10:58:35: kjmeagher commented:

> Do you want PFilt files or Level2 files for this?

**Comment by icecube-trac** [comment]
Trac update at 20151021T14:30:10: david.schultz commented:

> Either, though the frames need to be grabbed with the original release to preserve the serialization.

**Comment by icecube-trac** [comment]
Trac update at 20151021T14:47:22: kjmeagher commented:

> I was offering to be helpful and recommend which files to use from old detector configurations, do you want to go all the way back to IC9? you can figure out the grabbing them part

**Comment by icecube-trac** [comment]
Trac update at 20151021T14:50:24: david.schultz commented:

> I can probably figure out to IC79 and newer, since I worked on those. If you can give me a file list for previous years, that would be helpful (yes, I think we want to go all the way back).

**Comment by icecube-trac** [comment]
Trac update at 20151022T13:11:25: kjmeagher commented:

> midlevel
> IC9 : /data/ana/PointSource/IC9_PS/atmnu-analysis-data/reco_2_5_multiseed_data/paraboloid.level3.Run00088339.i3.gz
> IC22: /data/exp/IceCube/2007/filtered/level2/0531/PFFilt_PhysicsTrig_PhysicsFiltering_Run00107959_Level2_Part00000000.i3.gz
> IC40: /data/exp/IceCube/2008/filtered/level2_fix/0406/Level2_All_Run00110782_Part00000000.i3.gz
> IC59: /data/exp/IceCube/2009/filtered/level2/0520/Level2a_IC59_data_Run00113823_Part00000000.i3.gz
> 
> PFFilt:
> IC9 : /data/exp/IceCube/2006/filtered/PFFilt/0602/PFFilt_PhysicsTrig_PhysicsFilt_Run00088339_00000.tar.gz
> IC22: /data/exp/IceCube/2007/filtered/GRB/0531/PFRaw_TestData_PhysicsFiltering_Run00107959_Subrun00000000_00000000.tar.gz
> IC40: /data/exp/IceCube/2008/filtered/PFFilt/0406/PFFilt_PhysicsTrig_PhysicsFiltering_Run00110782_Subrun00000000_00000000.tar.gz
> IC59: /data/exp/IceCube/2009/filtered/PFFilt/0520/PFFilt_PhysicsTrig_PhysicsFiltering_Run00113823_Subrun00000000_00000000.tar.gz
> 
> Note: I included the PFFIle file from IC9 for completeness but it is not a i3 file. PFFilt files from back then were raw DAQ files

**Comment by icecube-trac** [comment]
Trac update at 20151111T21:05:00: 
* david.schultz commented:

> I give up on IC40 and earlier for now. It really needs boost 1.33, because that's what all the code is designed around.
> 
> Trimmed data files for IC59+ available here:
> 
> ```text
> /data/user/dschultz/olddata/
> ```
> 
> Note that in IC86 we synced PFFilt and Level2 offline releases, so those years don't have PFFilt files.
> 
> Next Steps:
> 1. nega adds these to test data
> 2. A simple test is developed to test deserialization for:
>    - I3Frame
>    - I3EventHeader
>    - maybe an I3Particle


* david.schultz changed _comment0 from:

> I give up on IC40 and earlier for now. It really needs boost 1.33, because that's what all the code is designed around.
> 
> Trimmed data files for IC59+ available here:
> {{{
> /data/user/dschultz/olddata/
> }}}
> 
> Note that in IC86 we synced PFFilt and Level2 offline releases, so those years don't have PFFilt files.
> 
> Next Steps:
> 1. nega adds these to test data
> 2. A simple test is developed to test deserialization for:
>    - I3Tray
>    - I3EventHeader
>    - maybe an I3Particle

to:

> 1449776905442904

**Comment by icecube-trac** [comment]
Trac update at 20151209T22:26:51: 
* nega commented:

> Data exists in `${I3_TESTDATA}/dataio/olddata`


* nega changed status from "accepted" to "assigned"
* nega changed owner from "nega" to "kjmeagher"

**Comment by icecube-trac** [comment]
Trac update at 20160126T10:42:08: kjmeagher changed owner from "kjmeagher" to "david.schultz"

**Comment by icecube-trac** [comment]
Trac update at 20160126T21:42:18: 
* david.schultz changed status from "assigned" to "accepted"
* david.schultz commented:

> Oh, fine.  I'll do it.

**Comment by icecube-trac** [comment]
Trac update at 20160126T22:39:46: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 2390:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2390"
> test old data. closes #1400
> ```


* david.schultz changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:10: 
* olivas commented:

> Milestone offline V15-12-00 deleted


* olivas changed milestone from "offline V15-12-00" to ""

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:51:02: nwhitehorn commented:

> In changeset 169182:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169182"
> test old data. closes #1400
> ```
